src/components/Navigation/NavBar/index.jsx

```diff
@@ -3,18 +3,29 @@ import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 
+import { languages } from 'components/LanguageWrapper';
+import { preferencesSetLocale } from 'state/actions/preferences';
+import { useFormatMessage } from 'hooks';
+import en from 'assets/en.png';
+import es from 'assets/es.png';
 import { logout } from 'state/actions/auth';
 import paths from 'pages/Router/paths';
 import defaultLogo from 'assets/user-default-log.svg';
 import Link from '../Link';
 
+const flags = {
```

Comment:
It's possible to have this flags logic outside the component file? Maybe in `utils/`.